modules/ROOT/pages/sap/sap-connector.adoc

```diff
@@ -269,19 +272,38 @@ There are no special Advanced properties for this configuration.
 |Log trace flag |logTraceFlag |Indicates if the trace should be logged on the Mule server.|false
 |===
 
-== Use Case: Send an IDoc to SAP
+== Use Case: Function Source - Return Values to SAP
+
+The current use case describes how to create a Mule application that receives a function using *Function source* which passes the export parameter back to SAP.
+
+. Create a new *Mule Project* in Anypoint Studio.
+. Drag *Function source* to the flow.
+. Create a new *SAP Inbound* global element configuration and fill with environment values.
+. Add a transform message component to the flow and fill the details based on metadata. Example:
++
+image::sap/sap-function-return-response.png[]
++
+The flow should look like this:
++
+image::sap/sap-flow-response.png[]
++
+In the SAP Studio interface, you should see:
++
+image::sap/sap-result-sap-gui.png[]
+
+=== Send IDoc to SAP
 
-The use case describes how to create a Mule application to send an IDoc to SAP.
+This use case describes how to create a Mule application to send an IDoc to SAP.
```

Comment:
Avoid self-referential text where possible. Simply "To create a Mule app to send an IDoc to SAP:" saves the reader (and translators) time.